### PR TITLE
Prints environment in console during build

### LINKED ISSUE #7480: Add environment output when building
Hi, I would love to see what environment I'm building for when I run `ember b` or in general any script that will trigger run for me.

Something like this
```bash
user@bash>ember b
Building
Environment: development
.....
```

### PR DESCRIPTION
**Pull request by abhilashlr**
Fixes #7480

### CODE REVIEW

**Comment by abhilashlr** [comment]
@Turbo87 - Sorry, I missed adding you as a reviewer. Please review.

**Comment by kellyselden** [comment]
Would it be possible to post a screenshot?

**Comment by stefanpenner** [comment]
This seems good, but I believe I share @kellyselden thoughts, lets confirm the visuals look right.

**Comment by abhilashlr** [comment]
@kellyselden @stefanpenner 

Here's the screenshot:

image

File: tests/unit/tasks/build-watch-test.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ describe('build-watch task', function() {
 
       return runPromise.then(() => {
         expect(ui.output).to.include('cleaning up...');
+        expect(ui.output).to.include('Environment: test');
```
Comment (kellyselden) [inline tests/unit/tasks/build-watch-test.js]: Should we have an associated build-test?

Comment (abhilashlr) [reply tests/unit/tasks/build-watch-test.js]: I've just added that @kellyselden

**Comment by kellyselden** [comment]
Thanks!

**Pull request merged**